### DISCUSSION TITLE
Local text recognition using AI (OCR)

### DISCUSSION THREAD

**Discussion by I-I-IT** [ente-io/ente#1122] [Enhancements]
Text recognition for images is now available on all major platforms native gallery apps (Samsung, Apple, Windows).

It is super convenient since nowadays many websites limit copying functionality. It is also run locally.

Since Ente already supports more advanced AI features, this should be fairly easy.

**Comment by ua741** [comment]
Duplicate of https://github.com/ente-io/ente/discussions/547
